### ISSUE TITLE
Large number of toots causes failure to load

### ISSUE THREAD

**Issue by notagoat** [tuskyapp/Tusky#2246]
When setting my post count to 100000000000000000 via database manipulation, the web client functions fine but Tusky crashes, including people who've interacted with me. This number is half the max size of a bigint that is used in the database to store post count.
* * * *
- Tusky Version: 15.1
- Android Version: 9
- Android Device: moto e6 plys
- Mastodon instance (if applicable): hellsite.site

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by charlag** [comment]
I don't think we should invest time into this to be honest, it is, how do I say it "out of spec"
